requirements.txt

```diff
@@ -7,10 +7,10 @@ automat==0.6.0  # from twisted
 incremental==16.10.1  # from twisted
 jinja2==2.10
 markupsafe==1.1.0  # from jinja2
-pyopenssl==19.0.0
+pyopenssl>=21.0.0,<23.0.0
```

Comment:
In debian 10 there is python3-openssl in version 19.0.0 - do you know if it is really necessary to use >= 21.0.0?

Reply:
Ok: unit tests fail with pyopenssl 19.0.0 from pypi, but running the tests locally on the stack with the debian-provided 19.0.0 works. They probably patched it.